demos/motortwit/motortwit/templates/timeline.html

```diff
@@ -17,11 +17,11 @@ <h2>{{ self.title() }}</h2>
           This is you!
         {% elif followed %}
           You are currently following this user.
-          <a class=unfollow href="{{ url('unfollow_user', parts={'username': profile_user.username})
+          <a class=unfollow href="{{ url('unfollow_user', username=profile_user.username)
```

Comment:
it appears that the `url_for` does not accept the dict `parts` anymore, but instead it waits for `**kwargs`